db/migrate/20230919143651_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] @Kgomotso196 Great work adding reference to the user, however the user should be referenced using author_id, I would recommend you change your code to `add_reference :posts, :author, null: false, foreign_key: true` to fulfill project requirement
![image](https://github.com/Kgomotso196/Blog/assets/61408860/02320260-7609-4cda-bdaa-d4e2bc90abe3)

Reply:
Thank you for your constructive feedback @danmainah.